rules/python3.mk

```diff
@@ -0,0 +1,38 @@
+# python3.5.2+
```

Comment:
Removed.

---
In reply to: [99899345](https://github.com/Azure/sonic-buildimage/pull/245#discussion_r99899345) [](ancestors = 99899345)